### ISSUE TITLE
Website for croudsourced translating?

### ISSUE THREAD

**Issue by wofsauge** [wofsauge/External-Item-Descriptions#11]
random item + image + english description + language chooser

**Comment by ngregrichardson** [comment]
I know this is a year old, but is this something that would still be wanted?

**Comment by wofsauge** [comment]
> I know this is a year old, but is this something that would still be wanted?

If a website for that usecase exists or is easily buildable, then yes it would be nice to have! Only real issue with such a concept would be to ensure the quality of the submitted translations.

**Comment by ngregrichardson** [comment]
I was thinking for ensuring the quality, as well as giving descriptions to translate it would give the descriptions that other people have already translated and you can upvote/downvote them.

**Comment by wofsauge** [comment]
Sounds good 👍

**Comment by Ma-rcx** [comment]
Yeah, crowdin is not the best. I would love to use weblate, but I don't have a server to host it on.

**Comment by ngregrichardson** [comment]
I can play around with Weblate. I have a few servers we could put it on if it's viable, if not a basic custom solution would work. I'm trying to think about how we might want to automate updates...for example, it'd be very nice to have it pull needed translations from GitHub and make PRs once a translation is "approved".

**Comment by Ma-rcx** [comment]
I copied this idea from https://translate.jellyfin.org/, they have a nice setup. I don't have much experience with it, but it seems to be fairly extensive and customizable. I don't see this as overly urgent, so we have time to test and figure out an ideal solution.

**Comment by magagnato4106** [comment]
Regarding croudsourced translating, I know a game which allows users to translate games. The site in which is based is https://explore.transifex.com and I hope it could be of your help!

**Comment by wofsauge** [comment]
Closing issue as this will not be done anymore.
